pkg/util/uint128/uint128.go

```diff
@@ -53,6 +53,11 @@ func (u Uint128) Sub(n uint64) Uint128 {
 	return Uint128{hi, lo}
 }
 
+// Equal judge whether two Uint128 is equal.
+func (u Uint128) Equal(t Uint128) bool {
```

Comment:
For field `ChecksumID` of `ComputeChecksumRequest `, there is no `Equal` method for it.